homeassistant/components/freebox/home_base.py

```diff
@@ -77,19 +80,52 @@ async def async_update_signal(self):
             )
         self.async_write_ha_state()
 
-    async def set_home_endpoint_value(self, command_id: Any, value=None) -> None:
+    async def set_home_endpoint_value(self, command_id: Any, value=None) -> bool:
         """Set Home endpoint value."""
         if command_id is None:
             _LOGGER.error("Unable to SET a value through the API. Command is None")
-            return
-        await self._router.home.set_home_endpoint_value(
-            self._id, command_id, {"value": value}
-        )
+            return False
+        try:
+            await self._router.home.set_home_endpoint_value(
+                self._id, command_id, {"value": value}
+            )
+        except asyncio.TimeoutError as error:
+            _LOGGER.warning("The Freebox API Timeout during a value set %s", error)
+            return False
+        except TimeoutError:
+            _LOGGER.warning("The Freebox Timeout during a value set")
+            return False
+        except aiohttp.client_exceptions.ClientError as error:
+            _LOGGER.warning("The Freebox API ClientError during value set %s", error)
+            return False
+        return True
+
+    async def get_home_endpoint_value(self, command_id: Any) -> Any | None:
+        """Get Home endpoint value."""
+        if command_id is None:
+            _LOGGER.error("Unable to GET a value through the API. Command is None")
+            return None
+        try:
+            node = await self._router.home.get_home_endpoint_value(self._id, command_id)
+        except asyncio.TimeoutError as error:
+            _LOGGER.warning(
+                "The Freebox API Timeout during a value retrieval %s", error
+            )
+            return None
+        except aiohttp.client_exceptions.ClientError as error:
+            _LOGGER.warning(
+                "The Freebox API ClientError during a value retrieval %s", error
+            )
+            return None
+        except TimeoutError:
+            _LOGGER.warning("The Freebox Timeout during a value retrieval")
+            return None
```

Comment:
Those exceptions should be removed, use exceptions from the Freebox lib.

Reply:
Exception removed